app.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+//store hours
+var openHours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm'];
+
+var table = document.createElement('table'); //tableEl = <table></table> in HTML
+document.body.appendChild(table);
+
+var storeOne = new CookieStore('first-and-pike', 'First and Pike', 23, 65, 6.3);
+var storeSeaTac = new CookieStore('seatac-airport', 'Seatac Airport', 3, 24, 1.2);
+var storeSeattle = new CookieStore('seattle-center', 'Seattle Center', 11, 38, 3.7);
+var storeCapitol = new CookieStore('capitol-hill', 'Capitol Hill',20, 38, 2.3);
+var storeAlki = new CookieStore('alki', 'Alki', 2, 16, 4.6);
+
+//stores Array
+var stores = [storeOne, storeSeaTac, storeSeattle, storeCapitol, storeAlki];
+
+function CookieStore(name, displayName, minCustomers, maxCustomers, avgCookies){
+  this.name = name;
+  this.displayName = displayName;
+  this.minCustomers = minCustomers;
+  this.maxCustomers = maxCustomers;
+  this.avgCookies = avgCookies;
+  this.hourlyCount = [];
+  this.range = maxCustomers - minCustomers;
+};
+
+CookieStore.prototype.salesPerHour = function(){
+  return Math.floor(Math.random() * ((this.range + 1) + this.minCustomers) * this.avgCookies);
+};
+
+CookieStore.prototype.populateHourlyCount = function() {
+  for (var i = 0; i < openHours.length; i++) {
+    this.hourlyCount.push(this.salesPerHour());
+  }
+};
+CookieStore.prototype.makeRow = function() {
+  var storeRow = document.createElement('tr');
+  table.appendChild(storeRow);
+
+  var storeName = document.createElement('th');
+  storeRow.appendChild(storeName);
+  storeName.textContent = this.displayName;
+
+  for (var i = 0; i < this.hourlyCount.length; i++) {
+    var hourlyCookieCount = document.createElement('td');
+    storeRow.appendChild(hourlyCookieCount);
+    hourlyCookieCount.textContent = this.hourlyCount[i];
+  }
+};
+function displayTimes(){
+  for (var ii = 0; ii < openHours.length; ii++){
+    var timeOfDay = document.createElement('th');
+    timeOfDay.textContent = openHours[ii];
+    table.appendChild(timeOfDay);
+  }
+}
+
+var locations = 'Locations:';
+var totalHead = document.createElement('th');
+totalHead.textContent = locations;
+table.appendChild(totalHead);
+
+function populateStoreCount() {
+  for (var i = 0; i < stores.length; i++){
+    stores[i].populateHourlyCount();
+  }
+}
+
+function makeAllRows() {
+  for (var i = 0; i < stores.length; i++){
+    stores[i].makeRow();
+  }
+}
+
+displayTimes();
```

Comment:
You could even break these three lines into another function, like 'drawTable' or something!